sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/misc.scala

```diff
@@ -432,4 +433,35 @@ case class AesDecrypt(
     copy(newChildren(0), newChildren(1), newChildren(2), newChildren(3))
   }
 }
+
+@ExpressionDescription(
+  usage = """_FUNC_(expr) - Return the estimated distinct count associated with this HyperLogLog++ sketch.""",
```

Comment:
same, let's please add an example and diagram to show the protocol of the sketch buffer in here?